packages/app-elements/src/ui/atoms/Badge/Badge.tsx

```diff
@@ -6,12 +8,14 @@ export interface BadgeProps
   extends Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
   /** Render a different variant. */
   variant: keyof typeof variantCss
+  icon?: keyof typeof iconMapping
```

Comment:
what do you think if we do this instead?
```
  icon?: IconProps['name']
```

Reply:
alright! sounds cleaner!

Reply:
sounds cleaner! :) fixed!